spec/factories/authentication.rb

```diff
@@ -99,6 +99,22 @@
     authtype "metrics"
   end
 
+  factory :automation_manager_authentication, :parent => :authentication do
+    type "ManageIQ::Providers::AutomationManager::Authentication"
```

Comment:
Please use `factory :automation_manager_authentication, :parent => :authentication, :class => "ManageIQ::Providers::AutomationManager::Authentication"`

I'm not sure why, but this seems to be the only file that diverges from the pattern.
When setting `type`, initialization would go through the parent class, when setting `:class`, initialization should go through the subclass.